apps/experiments/models.py

```diff
@@ -283,6 +289,22 @@ class Experiment(BaseTeamModel):
     """
 
     objects = ExperimentObjectManager()
+    status = models.CharField(
```

Comment:
How are we keeping track of the "active" version? By default an experiment is in "Draft" and when we make a version it becomes "released", but we can end up with multiple released versions yes?